Makefile

```diff
@@ -43,6 +43,13 @@ DESTTARGET := $(abspath $(DESTBINDIR)/$(TARGET))
 DESTCONFDIR := $(DESTDIR)/$(SYSCONFDIR)/$(CCDIR)
 DESTCONFIG := $(abspath $(DESTCONFDIR)/$(CONFIG_FILE))
 
+PAUSEDIR := pause
+PAUSEBIN := pause
+PAUSEDESTDIR := $(abspath $(DESTDIR)/var/lib/clear-containers/runtime/bundles/pause_bundle/bin/$(PAUSEBIN))
+
+override LIBS +=
+override CFLAGS += -Os -Wall -Wextra -static
```

Comment:
I don't think it will, but I'm not sure we need `override` here.

I'd use the flags specified by the CC 2.1 runtime:

- https://github.com/01org/cc-oci-runtime/blob/master/Makefile.am#L145..L152

Some of those might not be strictly required currently, but it's safest to specify them so we are "future-proof".